src/pages/Profile.js

```diff
@@ -6,11 +6,19 @@ const Profile = () => (
   <div className={styles.mainContainer}>
     <section className={styles.missionsContainer}>
       <h1>My Missions</h1>
-      <MissionsProfile />
+      <div>
+        <ul>
+          <MissionsProfile />
+        </ul>
+      </div>
```

Comment:
First of all, great job! :star: 
Just a little tiny suggestion: I think it will make more sense to take advantage of HTML table component instead of using list!  But either way is fine!
:heart:  :heart: